modules/prometheus-exporter_oracledb/conf/00-heartbeat.yaml

```diff
@@ -0,0 +1,14 @@
+module: oracle
+name: heartbeat
+
+transformation: true
```

Comment:
you can remove this one, changing transformation has no effect on heartbeat so it is basically useless (but not risky neither)

Reply:
okay, transformation is removed.